ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -92,7 +92,7 @@ export const NetworkListMenu = ({ onClose }) => {
   const orderedNetworksList = useSelector(getOrderedNetworksList);
 
   const newOrderNetworks = () => {
-    if (orderedNetworksList.length === 0) {
+    if (orderedNetworksList?.length === 0) {
```

Comment:
If `orderedNetworksList` is null or undefined, won't this not enter the branch because undefined !== 0?  I'm wondering if we want to do something like:

```suggestion
    if ((orderedNetworksList?.length ?? 0) === 0) {
```

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/22109/commits/f26dddbe5c41dcb651493cf753813778f35a36b2